### ISSUE TITLE
loss.backward() take too long time 

### ISSUE THREAD

**Issue by chenhuan001** [ubisoft/ubisoft-laforge-ZeroEGGS#26]
we use GPU with v100 32G and batchsize=256 （GPU use 15% and 30G），but loss.backward() will take 30s one time。so we need 60h to run a epoch，it is too slow.
Could you please tell me the scale of machine resources and the time it takes you to train the model ， Thank you very much
